services/docker-compose.local.yml

```diff
@@ -129,9 +129,9 @@ services:
         - traefik.http.services.${SWARM_STACK_NAME}_webserver.loadbalancer.sticky.cookie.secure=false
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver_local.service=${SWARM_STACK_NAME}_webserver
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver_local.entrypoints=http
-        - traefik.http.routers.${SWARM_STACK_NAME}_webserver_local.rule=hostregexp(`{host:.+}`) && PathPrefix(`/dev/`)
+        - traefik.http.routers.${SWARM_STACK_NAME}_webserver_local.rule=PathPrefix(`/dev/`)
```

Comment:
I also dont undrestand why there are there. I assume they handled some corner-case for some type of requests, since if I read it correctly you are asserting that the "hostname" (`host`) is not an empty string with that regex. We need to remove it since traefik-v3 at least doesnt support this verbatim syntax anymore, and then observe and see if everything still works as expected